### PR TITLE
Feature/ParentDashboard

### PR DESCRIPTION
**Pull request by SandraCoburn**
# Description
Work in Progress:
Changed the name of HomeScreen -.less, .js and container components to ParentDashboard -.js, .less and ParentDashboardContainer. Added links to navigation items to connect with other components. Changed some styling in the less files. 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [X] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [X] No
- [ ] Not necessary

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by JLC6290 — DISMISSED** [review]

File: src/components/pages/ParentDashboard/ParentDashboard.less [hunk 1]
```diff
@@ -82,11 +85,11 @@ button a:hover {
...
 }
-.ant-layout + .ant-layout {
-  margin-top: 48px;
-}
-.ant-layout-content {
-  flex: auto;
-  min-height: 9px;
-  background-color: #fff;
-}
+// .ant-layout + .ant-layout {
```
Comment (Schrese) [inline src/components/pages/ParentDashboard/ParentDashboard.less:88]: Do we need this still?

File: src/index.js [hunk 2]
```diff
@@ -21,16 +21,13 @@ import { ExampleDataViz } from './components/pages/ExampleDataViz';
...
-
-
 ReactDOM.render(
-//  
-    <Router>
-      <React.StrictMode>
-        <App />
-       </React.StrictMode>
-     </Router>,
+  //
```
Comment (Schrese) [inline src/index.js:25]: Is this doing something special?

**Review by bramirez96 — DISMISSED** [review]
Looks good!

**Review by bramirez96 — APPROVED** [review]
Here's hoping that these changes will work out for us!

**Review by Nick-Ohman — APPROVED** [review]
looks great!

**Pull request merged**